examples/http_sse_datastar_hello.rs

```diff
@@ -109,11 +109,13 @@ async fn main() {
     graceful.spawn_task_fn(async move |guard| {
         let exec = Executor::graceful(guard.clone());
 
-        let app = Router::new_with_state(controller.clone())
-            .with_get("/", handlers::index)
-            .with_post("/start", handlers::start)
-            .with_get("/hello-world", handlers::hello_world)
-            .with_get("/assets/datastar.js", DatastarScript::default());
+        let app = datastar_assets(
+            Router::new_with_state(controller.clone())
+                .with_get("/", handlers::index)
+                .with_post("/start", handlers::start)
+                .with_get("/hello-world", handlers::hello_world),
```

Comment:
as requested, you'll just have the assets maually defined here, the js file as well as the sourcemap. For the sourcemap I would recommend to add a `#[cfg(debug_assertions)]` on top so that the source map is only included in debug builds (not in release/production builds). Good to do such things in examples.